22-discovery.ks

```diff
@@ -30,10 +30,7 @@ systemctl enable nm-prepare.service
 echo " * enabling required system services"
 systemctl enable ipmi.service
 systemctl enable foreman-proxy.service
-systemctl enable discovery-setup.path
-systemctl enable discovery-setup.service
-systemctl enable discovery-autostart.service
-systemctl enable discovery-register.service
```

Comment:
Hmmm setup and autostart are gone/renamed, but I still see usage here:

```
root/etc/systemd/system/discovery-start-extensions.service
4:After=basic.target network.target network-online.target nss-lookup.target discovery-setup.service

root/etc/systemd/system/discovery-register.service
4:After=basic.target network.target network-online.target nss-lookup.target discovery-autostart.service foreman-proxy.service
```

Can you delete these and also make sure register service starts?